spot_driver_cpp/launch/spot_image_publisher.launch.xml

```diff
@@ -0,0 +1,28 @@
+<launch>
+  <!-- Args for required parameters: -->
+  <arg name="address" />
+  <arg name="username" />
+  <arg name="password" />
+
+  <!-- Args for optional parameters: -->
+  <arg name="spot_name" default="" />
+  <arg name="image_quality" default="70.0"/>
+  <arg name="rgb_cameras" default="true"/>
+  <arg name="publish_rgb" default="true"/>
+  <arg name="publish_depth" default="true"/>
+  <arg name="publish_depth_registered" default="false"/>
```

Comment:
I think we should work to defaulting to publishing rgb and depth for all cameras and doing depth registered calculation on this side and publishing that.  I think that's inline with these defaults so just making a note :)